ethereum_history_api/circuits/lib/src/account_with_storage_test.nr

```diff
@@ -0,0 +1,15 @@
+mod account_serialize {
+    use crate::account_with_storage::{Account, ACCOUNT_SERIALIZED_LEN};
+    use crate::misc::arrays::sub_array_equals;
+    use crate::fixtures::mainnet::london::vitalik_balance::account::account;
+
+    #[test]
+    fn success() {
+        let serialized: [Field; ACCOUNT_SERIALIZED_LEN] = account.serialize();
```

Comment:
To consider: change test name from success to simple, as success seems as if there could be some failing test